src/client/containers/Profile/index.js

```diff
@@ -1,5 +1,166 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Loader from '../../components/Loader/Loader.component';
+import { useAuthentication } from '../../hooks/useAuthentication';
 
 export default function Profile() {
-  return <div>This is your private profile</div>;
+  const [profileData, setProfileData] = useState(null);
```

Comment:
Profile container was from the boilerplate, the new container for profile is UserProfilePage. As everything from the boilerplate is going to be deleted, this should be moved.

Reply:
That should be discusses yet!